tests/assets/graceful-error.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2020 New Relic Corporation.
+  PDX-License-Identifier: Apache-2.0
+-->
+<html>
+  <head>
+    <title>RUM Unit Test</title>
+    <script>
+      window.errorSeen = false;
+      window.someOutput = false;
+      window.onerror = function () {
+        window.errorSeen = true;
+      };
+    </script>
+    {init} {config} {loader} {script-injection}
+  </head>
+  <body>
+    this is a generic page that is instrumented by the JS agent
+    <script>
+      window.someOutput = true;
```

Comment:
This executes before the agent fully inits. If you want to be doubly sure, this should go in a `load` listener **+ some `setTimeout`**.

Reply:
```suggestion
      setTimeout(function(){window.someOutput = true;}, 1000)
```

Reply:
That's still before load happens, no?